examples/GANs/3D-IWGAN/eval.py

```diff
@@ -12,51 +12,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import argparse
-import json
-import numpy as np
+
 import os
+import argparse
 import torch
-from torch.autograd import Variable
-import torch.optim as optim
-from torch.utils.data import DataLoader
-import sys
-from tqdm import tqdm
 
-np.random.seed(0)
+from architectures import Generator
+import kaolin as kal 
+
+
 torch.manual_seed(0)
 torch.cuda.manual_seed(0)
 
 
-from architectures import Generator
-
-import kaolin as kal 
-"""
-Commandline arguments
-"""
 parser = argparse.ArgumentParser()
-parser.add_argument('-expid', type=str, default='GAN', help='Unique experiment identifier.')
-parser.add_argument('-device', type=str, default='cuda', help='Device to use')
-parser.add_argument('-batchsize', type=int, default=50, help='Batch size.')
-
+parser.add_argument('--logdir', type=str, default='log', help='Directory to log data to.')
+parser.add_argument('--expid', type=str, default='GAN', help='Unique experiment identifier.')
+parser.add_argument('--device', type=str, default='cuda', help='Device to use')
```

Comment:
'.' missing at the end of the `help` field

Reply:
Nice catch, fixed.